hatch_build_plumed.py

```
@@ -0,0 +1,358 @@
+import sys
+from pathlib import Path
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class PlumedBuildHook(BuildHookInterface):
+    """Build hook that compiles PLUMED and bundles it with the wheel."""
+
+    PLUGIN_NAME = "plumed"
+
+    def initialize(self, version: str, build_data: dict) -> None:
```

Ruff (0.14.4)

Unused method argument: `version`

(ARG002)